### PR TITLE
Gitlab pipelines: use images from the Spack organization

### PR DESCRIPTION
**Pull request by alalazo**
Modifications:
- [x] Use runners from https://github.com/spack/gitlab-runners

### CODE REVIEW

**Comment by alalazo** [comment]
@spackbot run pipeline

**Comment by scottwittenburg** [comment]
Getting the correct arch when pulling images is supposed to be automatic I guess, and that's the reason for removing the arch from the image name?  Looks like the gitlab runners got the correct version, so I assume that must be working as expected.

File: var/spack/repos/builtin/packages/openjpeg/package.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Openjpeg(CMakePackage):
     homepage = 'https://github.com/uclouvain/openjpeg'
     url = 'https://github.com/uclouvain/openjpeg/archive/v2.3.1.tar.gz'
 
+    version('2.4.0', sha256='8702ba68b442657f11aaeb2b338443ca8d5fb95b0d845757968a7be31ef7f16d')
```
Comment (scottwittenburg) [inline var/spack/repos/builtin/packages/openjpeg/package.py:22]: This seems strange to include in the change, is the branch based on something from another PR?

Comment (alalazo) [reply var/spack/repos/builtin/packages/openjpeg/package.py:22]: No, I just wanted to trigger a rebuild and check the workers were fine too.

Comment (alalazo) [reply var/spack/repos/builtin/packages/openjpeg/package.py:22]: > Getting the correct arch when pulling images is supposed to be automatic I guess, and that's the reason for removing the arch from the image name? Looks like the gitlab runners got the correct version, so I assume that must be working as expected.

Yes, the images are multi-arch. Currently we have `x86_64`, `ppc64le` and `aarch64` (if it ever comes up in the future). you can check the "OS/Arch" tab [here](https://github.com/spack/gitlab-runners/pkgs/container/e4s-ubuntu-18.04)

Comment (scottwittenburg) [reply var/spack/repos/builtin/packages/openjpeg/package.py:22]: Got it, all that makes sense, thanks.

**Review by scottwittenburg — APPROVED** [review]

**Pull request merged**